src/blood-center/models/blood-center-event.entity.ts

```diff
@@ -31,8 +38,9 @@ export class BloodCenterEventEntity {
   geo_coordinates: string;
```

Comment:
Do we want to store those in string? Maybe an object with { latitude: number, longitude: number } will be more suitable?

Reply:
In the db I've been planning to keep it as a string: `"latitude/longitude"` like `"54.35/18.64"` or something alike.
For validating incoming/outgoing requests I'd probably use an object as that would be easier than a custom regex - this will however be implemented later when working on this module as a part of DTO and not the Entity itself.
Other way of storing this in the DB would be to create a separate table and I'm not a fan of this idea or break the `geo_cordinates` column in to two separate ones.

Reply:
Ok, that works for me 👍